src/engraving/infrastructure/io/mscreader.cpp

```diff
@@ -426,13 +438,13 @@ StringList MscReader::XmlFileReader::fileList() const
     m_device->seek(0);
     XmlStreamReader xml(m_device);
     while (xml.readNextStartElement()) {
-        if ("files" != xml.name()) {
+        if (xml.name() != "files") {

```

Comment:
why this (and the following)?

Reply:
well a) there's no good reason to put it in that order in the first place, as it reads very unnaturally, and prevents nothing, and b) it wasn't compiling due to a name hiding problem with operator != (though I'm not 100% certain why swapping the args resolves this)

Reply:
It did work before the PR though, didn't it?

Reply:
yes but a header file was introduced that introduced an operator!= in the mu::engraving namespace, hence the name hiding.